<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="12081" started-at="2020-07-21T14:57:31Z" finished-at="2020-07-21T14:57:43Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="12081" started-at="2020-07-21T14:57:31Z" finished-at="2020-07-21T14:57:43Z">
      <class name="UnbreakableTestTrial">
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:UnbreakableTestTrial@1df90b0]" name="beforeMethod" is-config="true" duration-ms="7306" started-at="2020-07-21T14:57:31Z" finished-at="2020-07-21T14:57:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void UnbreakableTestTrial.unbreakableTest() throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@9306d3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="unbreakableTest()[pri:0, instance:UnbreakableTestTrial@1df90b0]" name="unbreakableTest" duration-ms="3154" started-at="2020-07-21T14:57:38Z" finished-at="2020-07-21T14:57:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[A detailed report will be created at 'D:\workspace\recheckweb4\autoheal-recheckweb\target\test-classes\retest\recheck\UnbreakableTestTrial.report'. You can review the details by using our CLI (https://github.com/retest/recheck.cli/) or GUI (https://retest.de/review/).

1 check(s) in 'UnbreakableTestTrial' found the following difference(s):
Test 'beforeMethod' has 2 difference(s) in 1 state(s):
trial resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: A detailed report will be created at 'D:\workspace\recheckweb4\autoheal-recheckweb\target\test-classes\retest\recheck\UnbreakableTestTrial.report'. You can review the details by using our CLI (https://github.com/retest/recheck.cli/) or GUI (https://retest.de/review/).

1 check(s) in 'UnbreakableTestTrial' found the following difference(s):
Test 'beforeMethod' has 2 difference(s) in 1 state(s):
trial resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
at de.retest.recheck.RecheckImpl.capTest(RecheckImpl.java:188)
at UnbreakableTestTrial.unbreakableTest(UnbreakableTestTrial.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- unbreakableTest -->
        <test-method status="FAIL" signature="afterMethod(java.lang.reflect.Method, [Ljava.lang.Object;)[pri:0, instance:UnbreakableTestTrial@1df90b0]" name="afterMethod" is-config="true" duration-ms="1585" started-at="2020-07-21T14:57:41Z" finished-at="2020-07-21T14:57:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void UnbreakableTestTrial.unbreakableTest() throws java.lang.Exception]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.Object;@9306d3]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[A detailed report will be created at 'D:\workspace\recheckweb4\autoheal-recheckweb\target\test-classes\retest\recheck\UnbreakableTestTrial.report'. You can review the details by using our CLI (https://github.com/retest/recheck.cli/) or GUI (https://retest.de/review/).

4 check(s) in 'UnbreakableTestTrial' found the following difference(s):
Test 'beforeMethod' has 2 difference(s) in 4 state(s):
00 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
01 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
02 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
03 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: A detailed report will be created at 'D:\workspace\recheckweb4\autoheal-recheckweb\target\test-classes\retest\recheck\UnbreakableTestTrial.report'. You can review the details by using our CLI (https://github.com/retest/recheck.cli/) or GUI (https://retest.de/review/).

4 check(s) in 'UnbreakableTestTrial' found the following difference(s):
Test 'beforeMethod' has 2 difference(s) in 4 state(s):
00 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
01 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
02 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
03 resulted in:
Metadata Differences:

input at 'html[1]/body[1]/main[1]/form[1]/input[1]':
id: expected="username123", actual="username123456"
input at 'html[1]/body[1]/main[1]/form[1]/input[2]':
id: expected="password", actual="password123"
at de.retest.recheck.RecheckImpl.capTest(RecheckImpl.java:188)
at de.retest.recheck.RecheckImpl.cap(RecheckImpl.java:200)
at de.retest.web.selenium.AutocheckingRecheckDriver.cap(AutocheckingRecheckDriver.java:88)
at de.retest.web.selenium.AutocheckingRecheckDriver.quit(AutocheckingRecheckDriver.java:108)
at UnbreakableTestTrial.afterMethod(UnbreakableTestTrial.java:74)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- UnbreakableTestTrial -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
